src/AdminDashboard/Api/Glpi.js

```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch'
+const URL = "https://dev.flyve.org/glpi/apirest.php/"
+// const URL = "http://glpis42.local/apirest.php/"
+const userToken = "zq6xxHdUwJfh1pnTcKi66sLvX978KKGVvOFK9LBS"
```

Comment:
Try to use an external config file

Reply:
Hi @ajsb85 
Yes, this is already designed to be used in the library